utils/codepointtrie/src/codepointtrie.rs

```diff
@@ -198,10 +203,12 @@ impl<'trie, T: TrieValue> CodePointTrie<'trie, T> {
         // - The `ZeroVec` serializer stores the length of the array along with the
         //   ZeroVec data, meaning that a deserializer would also see that length info.
 
+        let error_value = data.last().unwrap_or(T::DEFAULT_ERROR_VALUE);
```

Comment:
Nit: Do one of the following

1. Add `error_value` as an argument to `try_new`
2. Return an Error since this function is fallible